script/build

```diff
@@ -14,7 +14,9 @@ windows=
 
 setup_gopath() {
   TMPDIR="${LOCALAPPDATA:-$TMPDIR}"
-  TMP_GOPATH="${TMPDIR:-/tmp}/go"
+  MY_TMPDIR=${TMPDIR:-/tmp}
+  # TMPDIR may or may not have had a trailing slash.
```

Comment:
There is no need for these code comments. It should be evident that we're stripping a trailing slash.